JS/Exercise_21/index.js

```diff
@@ -0,0 +1,6 @@
+alert('Enter two numbers to see if the first is greater than the second');
+
+let firstNumber = prompt('Enter the first number');
+let secondNumber = prompt('Enter the second number');
+
+confirm(firstNumber > secondNumber);
```

Comment:
Hey Daniel 
it would be nice to convert entered values to numbers.
And then compare them.